### PR TITLE
(DOCSP-17242): BI Connector Landing Page

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
[JIRA](https://jira.mongodb.org/browse/DOCSP-17242)
[Stage](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17242/)

moved existing landing page content to [What is BI Connector page](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17242/what-is-the-bi-connector/)

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -1,129 +1,144 @@
...
-   :figwidth: 700px
-   :align: center
+         .. step:: Create visualizations of MongoDB data with Tableau
 
-On Premises Database and |bi-short|
------------------------------------
+            Many business intelligence visualization tools can't
+            communicate with MongoDB databases using MongoDB Query
+            Language. |bi| translates MQL into SQL, enabling
+            communication with any BI tool that uses |odbc| connections.
```
[the remark this answers is not in the document]
Comment (sposetti) [reply source/index.txt:72]: cc @jw

File: source/what-is-the-bi-connector.txt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+Traditional business intelligence tools work with flat, tabular data.
+These tools aren't sophisticated enough to understand three-dimensional
+data stored in MongoDB databases.
+
+The MongoDB Connector for Business Intelligence (BI) allows you to
+create queries with |sql| to visualize, graph, and report on your
+three-dimensional MongoDB Enterprise data using relational 
+business intelligence tools such as :doc:`Tableau </connect/tableau>`,
+:doc:`MicroStrategy </connect/microstrategy>`, and
+:doc:`Qlik </connect/qlik>`.
```
[the remark this answers is not in the document]
Comment (jwilliams-mongo) [reply source/what-is-the-bi-connector.txt]: updated

File: source/index.txt [hunk 3]
```diff
@@ -1,129 +1,144 @@
...
 
-.. default-domain:: mongodb
+.. introduction::
 
-About the |bi|
---------------
+   Traditional business intelligence tools are designed to work with
+   tabular, row-and-column data. The MongoDB Connector for BI allows you
+   to analyze rich, document-oriented MongoDB data with familiar
+   applications such as Tableau, Qlik, and Excel.
```
[the remark this answers is not in the document]
Comment (jwilliams-mongo) [reply source/index.txt:15]: updated

**Comment by jwilliams-mongo** [comment]
Hi @mango-db if you have a minute can you review this PLP PR? I'd like to merge this today if possible. Thank you!

**Review by mango-db — APPROVED** [review]
LGTM from an engineering perspective, though I'm still seeing a reference to Qlik in `index.txt` - not sure if that still needs to be removed. 

[Staging](https://docs-mongodborg-staging.corp.mongodb.com/DOCSP-17242/bi-connector/cgoecknerwald/master/)

**Pull request merged**